src/adapter/something.js

```diff
@@ -0,0 +1,9 @@
+function _generatedCode(defaultValue) {
+  try {
+    return (function (def) {
+      return global.customDebuggerDescription(this, def)
+    })(defaultValue);
+  } catch (e) {
+    return e.stack || e.message || String(e);
+  }
+}
```

Comment:
did you mean to check this and src/adapter/notes.js in?

Reply:
Removed